accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -287,74 +287,50 @@ applicable to Purchase and Sales journals that can be related to the
 different sets of document types available in Chile. By default, all the sales journals created will
 use documents. 
   
-.. image:: media/Chile10.png
+.. image:: media/Chile41.png
    :align: center
    :alt: Document type configuration on Journal.
-
-Sequences
-*********
-
-When a Journal is created we have two options with document types sequences. 
-
-1. Create new sequences: In this case, Odoo will create new sequences for the most common document
-   types. 
-2. Use existent sequence: If the sequence should be shared between several journals, the user can
-   add manually sequences previously created.  
-
-.. image:: media/Chile11.png
-   :align: center
-   :alt: Steps to add new sequences on journals.
    
 .. important::
    For the Chilean localization it’s important to define the default Debt and Credit accounts 
    as they are required for one of the debit notes use cases.
    
 CAF
-***
+---
 
-For each sequence related to the documents type, a CAF (Folio Authorization Code) is required,
-the CAF is a file the SII provides to the Emisor with the folio/sequence authorized for the 
-electronic invoice documents.
+A CAF (Folio Authorization Code) is required, for each document type that is issued to your
+customers, the CAF is a file the SII provides to the Emisor with the folio/sequence authorized 
+for the electronic invoice documents.
 
 Your company can make several requests for folios and obtain several CAFs, each one associated
-with different ranges of folios. In this way, you could order folios per the different
-segmentations you have in your journals (example per office) thus avoiding distributing a range
-of folios. In this way, each branch would have its private key of the respective CAF.
+with different ranges of folios. In V14 the CAF are shared within all the journals, this means
+that you only need one active CAF per document type and it will be applied on all journals.
 
 Please refer to the SII documentation to check the detail on how to acquire the CAF:
 `SII Timbraje <https://palena.sii.cl/dte/mn_timbraje.html>`_.
 
 	
 Configuration
-^^^^^^^^^^^^^
-
-Once you have the CAF files you need to associate them with journal sequence in Odoo,
-after you create the sequence for the first time the will be in red, which indicates
-that no CAF has been configured for that particular sequence:
-
-.. image:: media/Chile12.png
-   :align: center
-   :alt: Sequences per document type on journals.
-
-In order to add a CAF, just follow these steps:
+~~~~~~~~~~~~~
 
-1. Open the sequence. 
-2. Go the the Tab “CAF”.
-3. Upload the file.
-4. Save the CAF.
-5. Save the sequence. 
+Once you have the CAF files you need to associate them with a document type in Odoo,
+in order to add a CAF, just follow these steps:
+ 
+1. Access to :menuselection:`Accounting --> Settings --> CAF`
+2. Upload the file.
+3. Save the CAF.
 
-.. image:: media/Chile13.gif
+.. image:: media/Chile39.gif
```

Comment:
This file is WAY too large (3.5 MB) in comparison with its added value.
In git, a repository can only grow in size, never shrink. We must be careful not to add unnecessary large files as every single update made to those files in fact duplicates them. This doesn't scale at all and we are still paying the price of such mistakes in the odoo/odoo repository whose weight is now measured in GB.

Reply:
Got it, I removed it and switched it for a png image.